### PR TITLE
Add script to run migratation

### LINKED ISSUE #37: Deployment: Ensure App Deployment - develop branch for Milestone 1 on Heroku
## User story

**Who: As a** developer
**What: I want to be** able to see the development version online on https://guide-it-staging.herokuapp.com/
**Why: so that I can** test it in **DEVELOPMENT** before is goes to Production

## Acceptance criteria

Make sure that every **PR builds** on Develop and that the **Develop Heroku DB** is up to date and with the correct credentials, and both Migrations and Seeding are updated in this environment. 

image

## Implementation details

Check the env variables on https://dashboard.heroku.com/apps/guide-it-staging/settings and **DO NOT SHARE or COMMIT any information about Users or Passwords.**

### PR DESCRIPTION
**Pull request by sofiiadidovych**
# Description

I added db:migrate script to the run command. This will run db migration on each build. As far as I understand, we don't need to run seeds every time since it will delete all existing data.
Also I added environmental variables to Config Vars on Heroku.

Fixes #37

# How to test?

Every time start command runs on Heroku - all migrations should be automated.

# Checklist

- [x] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [x] This PR is ready to be merged and not breaking any other functionality

### CODE REVIEW

**Comment by Enqira** [comment]
When you run knex migrate:latest is going to create new tables, that's why I think that you need to run knex seed too to populate those tables, otherwise they will be empty. Just to be sure, bring this question in sunday class.

**Review by Enqira — APPROVED** [review]

File: package.json [hunk 1]
```diff
@@ -9,12 +9,14 @@
...
     "client": "webpack-dev-server --mode development --devtool inline-source-map --hot",
     "server": "nodemon ./src/server/bin/server",
     "dev": "concurrently \"npm run server\" \"npm run client\"",
     "db:clean": "cd ./src/server/ && knex migrate:rollback",
-    "db:setup": "cd ./src/server/ && knex migrate:latest && knex seed:run",
+    "db:setup": "npm run db:migrate && npm run db:seed",
+    "db:migrate": "cd ./src/server/ && knex migrate:latest",
+    "db:seed": "cd ./src/server/ && knex seed:run",
     "db:reset": "npm run db:clean && npm run db:setup",
     "storybook": "cd ./src/client/ && start-storybook -p 3007",
```
Comment (arya56) [inline package.json:21]: @sofiiadidovych Regarding setup database in Herku the following script should be added as well.`"heroku-postbuild": "npm run db:setup"`

File: package.json [hunk 2]
```diff
@@ -9,12 +9,14 @@
   },
   "scripts": {
     "build": "webpack --mode production",
-    "start": "npm run build && node ./src/server/bin/server",
+    "start": "npm run db:migrate && npm run build && node ./src/server/bin/server",
```
Comment (arya56) [inline package.json:12]: Better not to change The "start" script. Probably no need to do migration here

Comment (sofiiadidovych) [reply package.json]: Thank you, I fixed this.

File: package.json (hunk 1 above)
Comment (sofiiadidovych) [reply package.json:22]: @arya56 Thank you for your review. 
I added "heroku-postbuild" script. As far as I understand, we should only run migration script without seeding. The problem with the seeding is that it runs all the seed files every time, meaning we will delete all the existing data and insert the new data after every deployment.

Comment (sofiiadidovych) [reply package.json:22]: @dpfernandes thank you for your clarification on this issue. @arya56 I changed my "heroku-postbuilt" command as you asked: "heroku-postbuild": "npm run db:setup"
Hope this PR is ready for merging :)

**Review by dpfernandes — APPROVED** [review]
👍

**Pull request merged**